source/d3d11/draw_call_tracker.cpp

```diff
@@ -1,105 +1,168 @@
 #include "draw_call_tracker.hpp"
 #include "runtime.hpp"
 #include <math.h>
+#include "log.hpp"
 
 namespace reshade::d3d11
 {
 	void draw_call_tracker::merge(const draw_call_tracker& source)
 	{
-		_counters.vertices += source.vertices();
-		_counters.drawcalls += source.drawcalls();
+		_global_counter.vertices += source.total_vertices();
+		_global_counter.drawcalls += source.total_drawcalls();
 
-		for (auto source_entry : source._counters_per_used_depthstencil)
+		for (auto &[depthstencil, snapshot] : source._counters_per_used_depthstencil)
```

Comment:
This can be `for (const auto &[depthstencil, snapshot] : source._counters_per_used_depthstencil)` I think. Same for a couple of places below. I prefer to always make things `const` if possible.